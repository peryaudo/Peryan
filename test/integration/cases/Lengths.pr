var foo :: [Int] = [1, 1, 4, 5, 1, 4]
var bar :: String = "1234567"
var baz = [Bool](100, true)

mes String(foo.length)
mes String(bar.length)
mes String(baz.length)
