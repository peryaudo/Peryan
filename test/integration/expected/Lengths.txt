6
7
100
